RNTester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -750,6 +751,26 @@ class AnnounceForAccessibility extends React.Component<{}> {
   }
 }
 
+class SetAccessibilityFocus extends React.Component<{}> {
+  _handleOnPress = () => {
+    if (findNodeHandle(this.focusRef.current)) {
+      const reactTag = findNodeHandle(this.focusRef.current);
+      AccessibilityInfo.setAccessibilityFocus(reactTag);
+    }
+  };
+  render() {
+    this.focusRef = React.createRef();
```

Comment:
I don't think you normally want to create the ref in render, that would wipe it away every time render is called. Normally this would be done in the constructor.

Since you should change this to hooks this problem will go away on its own anyways. :)